server/resolvers/TestPlanReport/issuesResolver.js

```diff
@@ -54,9 +75,12 @@ const issuesResolver = async testPlanReport => {
                 isCandidateReview: hiddenIssueMetadata.isCandidateReview,
                 feedbackType,
                 isOpen: state === 'open',
-                testNumberFilteredByAt: hiddenIssueMetadata.testRowNumber
+                testNumberFilteredByAt: hiddenIssueMetadata.testRowNumber,
+                at,
+                browser
```

Comment:
Similar to my concern above, if a GitHub issue's AT or Browser name is edited and it doesn't match against any known `At` or `Browser`, this query would fail because those 2 fields are required. Can we guard against this here? Maybe we just don't want to have them be required and default to `N/A`, in those columns on the frontend and share that this is a hint that the test structure in the issue may be malformed to issue authors? Or perhaps we'd want to just not return this item?